TESTING.md

```diff
@@ -22,10 +22,12 @@ To run all sanity tests: `ansible-test sanity --docker -v`
 
 ## Running nox based sanity tests and code formatting
 
+These tests require [`nox`](https://pypi.org/project/nox) and [`antsibull-nox`](https://pypi.org/project/antsibull-nox/). Note that running `nox` without `antsibull-nox` present will automatically create a venv that installs all needed requirements. You can also use `pipx run noxfile.py` or similar Python runners; they detect the script metadata in `noxfile.py` and will automatically install `nox` and `antsibull-nox`.
```

Comment:
Probably worth noting that this only works with the latest nox version.

```suggestion
These tests require [`nox`](https://pypi.org/project/nox) and [`antsibull-nox`](https://pypi.org/project/antsibull-nox/). Note that running `nox` (version 2025.02.09 or greater) without `antsibull-nox` present will automatically create a venv that installs all needed requirements. You can also use `pipx run noxfile.py` or similar Python runners; they detect the script metadata in `noxfile.py` and will automatically install `nox` and `antsibull-nox`.
```

Reply:
Actually, both automatic installation and pipx support require newer nox, the latter because nox.main() didn't exist before, so that suggestion needs to be amended.

Reply:
I've moved the version bound to the first mention of nox.